__tests__/app.test.js

```diff
@@ -233,4 +233,116 @@ describe('GET /api/reviews/:review_id', () =>
             });
         });
     });
+});
+
+describe('PATCH /api/reviews/:review_id', () =>
+{
+    test('server responds with status - 200 and an updated review object for given positive number', () =>
+    {
+        return request(app)
+        .patch('/api/reviews/1')
+        .send({
+            inc_votes: 1
+        })
+        .expect(200)
+        .then((response) => 
+        {
+            const { review } = response.body;
+
+            expect(review[0].votes).toBe(2);
```

Comment:
we want the single review object not an array containing the object on the key of review